src/Model/GiftCardConfiguration.php

```diff
@@ -110,7 +110,7 @@ public function setBackgroundImage(?GiftCardConfigurationImageInterface $image):
     }
 
     /**
-     * {@inheritDoc}
+     * {@inheritdoc}
```

Comment:
We have kinda convention to not use phpdocs like `inheritdoc`.

Reply:
Shall I just remove the whole docblock ?

Reply:
Yes :)